proposals/etcd.adoc

```diff
@@ -0,0 +1,104 @@
+== etcd Project Proposal
+
+*Name of Project*: etcd
+
+*Description*:
+
+etcd is a consistent distributed key-value store, designed to hold small
+amounts of data that can fit entirely in memory (although etcd still writes
+to disk for durability) and mainly used as a separate coordination service
+for other distributed systems like Kubernetes. A typical etcd cluster is
+distributed over 3 to 5 nodes, for high availability, while it prioritizes
+consistency and partition tolerance. Which means, it provides one logical
+cluster view of many physical servers. So long as a majority of cluster is up,
+etcd continues to work, even under machine failures. This redundancy provides
+fault tolerance.
+
+etcd server implements Raft consensus algorithm for data replication. Raft is
+a leader-based protocol. Data is replicated from leader to follower; follower
+forwards proposals to leader, and leader decides what to commit or not. Leader
+persists and replicates an entry, once it has been agreed by the quorum of
+cluster. Therefore, adding more members to cluster would lead to more
+communication overhead and limited write scalability. Write (and consistent
+read) throughput is limited to durable write throughput of a single leader
+node.
+
+The underlying storage layer for Raft log is write-ahead log (WAL).
+Committed entries are written out to disk, so they can be replayed on restart.
+etcd uses gRPC for transport layer. Client employs HTTP/2 Ping for server
+health checking, and implements automatic failover under faulty networks.
+
+*Sponsor / Advisor from TOC*: Brian Grant <briangrant@google.com>
+
+*Unique Identifier*: etcd
+
+*License*: https://github.com/coreos/etcd/blob/master/LICENSE[Apache License 2.0]
+
+*Maturity Level*: Incubating
+
+*Source control repositories*: https://github.com/etcd-io (to be moved
+from https://github.com/coreos/etcd/issues/9965[August 27, 2018])
+
+*Issue tracker*: https://github.com/coreos/etcd/issues
+
+*Initial committers*: https://github.com/philips[Brandon Phillips] and
+https://github.com/xiang90[Xiang Li] at CoreOS.
+See https://github.com/coreos/etcd/blob/master/MAINTAINERS[MAINTAINERS].
+
+*Infrastructure requirements (CI / CNCF Cluster)*: N/A
+
+*External dependencies*: https://github.com/coreos/etcd/blob/master/bill-of-materials.json
+
+*Statement on alignment with CNCF mission*: etcd helps adopt cloud native
```

Comment:
Maybe @philips could help improve the wording here

Reply:
@bgrant0607 how about this?

A consistent and partition tolerant datastore, like etcd, is a base dependency for many cloud native architectures. They hold on to critical cluster configuration and locks while providing guarantees against individual machine failure, network partitions, or data center power loss. In the literature and ecosystem etcd, or similar systems (e.g. chubby, zookeeper, etc), provide the persistence layer for applications like Kubernetes, CoreDNS, Vitess, Borg, Mesos, and countless others.

Reply:
LGTM, thanks

Reply:
@philips @bgrant0607 Just updated this part. Thanks!